test-data/unit/check-selftype.test

```diff
@@ -272,18 +272,18 @@ from typing import Any, Generic, Self, TypeVar, overload
 T_co = TypeVar('T_co', covariant=True)
 
 class Config(Generic[T_co]):
-	@overload
-	def get(self, instance: None) -> Self: ...
-	@overload
-	def get(self, instance: Any) -> T_co: ...
-	def get(self, *a, **kw): ...
+    @overload
+    def get(self, instance: None) -> Self: ...
+    @overload
+    def get(self, instance: Any) -> T_co: ...
+    def get(self, *a, **kw): ...
```

Comment:
I'm guessing this was previously tab-indented and your editor auto-fixed it to spaces?

Reply:
Yeah, when I realized that I had converted the entire file to crlf endings locally (by using `--update-data`) and tried to fix that, my editor did this. Now it matches with the rest of the file so that's nice.